kumascript/src/lib/css-syntax.ts

```diff
@@ -347,8 +351,8 @@ export async function getCSSSyntax(
           `<a href="${valueDefinitionUrl}#${info.fragment}" title="${info.tooltip}">[</a>`
         );
         name = name.replace(
-          /\]$/,
-          `<a href="${valueDefinitionUrl}#${info.fragment}" title="${info.tooltip}">]</a>`
+          /\]([^']?)$/,
```

Comment:
Is this causing this change:
<img width="640" alt="image" src="https://github.com/mdn/yari/assets/495429/9e5261e5-5637-4b1b-8b17-baa8367a4b12">

If this is intended, can we also fix it for the opening bracket?

Reply:
As discussed, this change avoids the linking of the literal `]`, just the diff is outdated.